### ISSUE TITLE
Build crash when starting capture

### ISSUE THREAD

**Issue by Gabana** [RenderHeads/UnityPlugin-AVProMovieCapture#153]
Hi! One of my users has experienced a weird crash on a build that seems related to AVProMovieCapture. I´m affraid i cannot reproduce the steps myself but seems like there´s a problem with the audio source?

Here´s a video of the crash:
https://onedrive.live.com/?authkey=...E1BEB!1174&parId=C0FA90FB584E1BEB!181&o=OneUp

And the last few lines of the log:
```
[AVProMovieCapture] Start File Capture: 1080x1080 @ 30.00fps [RGBA32] vcodec:'H264' audio source:'Universal Audio Thunderbolt' acodec:'AAC' to file: '/Users/mattmitchell/Dropbox/Blank Tempo/Volta/Exports/VoltaClip_QEWY_2022-01-18_20-57-17_1080x1080.mp4'
2022-01-18 20:57:17.306 Volta Create[18784:2104206] *** Terminating app due to uncaught exception 'NSInvalidArgumentException', reason: '*** -[AVAssetWriterInput initWithMediaType:outputSettings:sourceFormatHint:] AudioChannelLayout channel count must be positive'
*** First throw call stack:
(
    0   CoreFoundation                      0x00007fff379b24d7 __exceptionPreprocess + 250
    1   libobjc.A.dylib                     0x00007fff7052b5bf objc_exception_throw + 48
    2   AVFoundation                        0x00007fff33205eb2 -[AVAssetWriterInput dealloc] + 0
    3   AVFoundation                        0x00007fff33205919 +[AVAssetWriterInput assetWriterInputWithMediaType:outputSettings:] + 40
    4   AVProMovieCapture                   0x0000000127f4c50f AVPMC_EncodeMonoToAmbisonic + 10018
    5   AVProMovieCapture                   0x0000000127f449de UnityPluginUnload + 2489
    6   AVProMovieCapture                   0x0000000127f4836e AVPMC_CreateRecorderVideo + 460
    7   ???                                 0x000000015e55aab4 0x0 + 5877639860
    8   ???                                 0x000000015e55a323 0x0 + 5877637923
    9   ???                                 0x000000015e558443 0x0 + 5877630019
)
libc++abi.dylib: terminating with uncaught exception of type NSException
Obtained 0 stack frames.
```

Computer specs: MacBook Pro 2019 16" with 32GB of RAM, 2TB SSD, 2.4 GHz Intel Core i9 Processor and Intel UHD Graphics 630 1536 MB.

Any ideas why this could happen?

Thanks in advance!
D

**Describe the bug**

**Your Setup (please complete the following information):**
 - Unity version: 
 - AVPro Movie Capture version: 
 - Operating system version: 
 - Capture component used: 
 - Capture settings (resolution, frame-rate, codec): 

**To Reproduce**

**Logs**
If applicable, add error logs to help explain your problem.

**Screenshots**

**Videos**
If applicable, add a copy of your video or the URL

**Comment by AndrewRH** [comment]
Yes "AudioChannelLayout channel count must be positive" from the log does seem to indicate something related to the audio.

Which version of AVPro Movie Capture are you using?

**Comment by MorrisRH** [comment]
Would be good to know what kind of device is being used to capture from, I'm not familiar with "Universal Audio Thunderbolt".

**Comment by MorrisRH** [comment]
This is due to us adding support for multi-channel capture from device in version 4.7.5. I'll add some checks around that value but without knowing more about the device it's going to limit capture to a single channel for your user.

**Comment by MorrisRH** [comment]
A fix will be in the next release. I'm somewhat blind on this though as I don't have access to the device in question.

**Comment by Gabana** [comment]
Thanks @MorrisRH. We are using version 4.7.2. The  "Universal Audio Thunderbolt" is my user´s audio interface, although that probably is not enough information. I asked him for details.
When he uses Blackhole (https://github.com/ExistentialAudio/BlackHole), it works fine

**Comment by Gabana** [comment]
This is the audio interface he is using:
https://www.uaudio.com/audio-interfaces/apollo-8.html

**Comment by AndrewRH** [comment]
4.7.7 has now been released with the fix for this issue
